opensanctions/metadata/us_congress.yml

```diff
@@ -0,0 +1,29 @@
+entry_point: opensanctions.crawlers.us_congress
+title: US Congress PEPs
+prefix: us_congress
+publisher:
+  name: Congress
+  country: us
+  url: https://api.congress.gov/
+  official: true
+url: https://api.congress.gov/
+data:
+  url: https://api.congress.gov/v3/member/
+  format: JSON
+
+lookups:
+  position:
+    normalize: true
+    options:
+      - match: House of Representatives
+        name: Member of the United States House of Representatives
+        topics:
+          - role.pep
+          # - role.scope.nat
+          # - role.gov.legis
+      - match: Senate
+        name: Member of the United States Senate
+        topics:
+          - role.pep
+          # - role.scope.nat
+          # - role.gov.legis
```

Comment:
this is cute but pepdb has 121664 positions so this approach won't scale to that

Reply:
Not with `yaml.safe_load` for sure :-P